helm-charts/landing/values.yaml

```diff
@@ -0,0 +1,57 @@
+# Default values for landing.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+
+# If provided, these will be used by the deployment
+# nodeSelector: XXXXXXX
+
+image:
+  # same tag for everyone as it is how landing repo Makefile works
+  tag: master
+  pullPolicy: IfNotPresent
+  repository: quay.io/srcd
```

Comment:
Nit-picking: Actually, the name of the repository is quay.io/srcd/landing, or quay.io/srcd/landing-api,.. I'd either use 3 different repository keys, or rename this to "registry".

Reply:
Also, do we really want to use 3 different repositories for these images? As an alternative, we could use "api" or "slacking" as part of the tag, together with the version, and this way we would have all 3 different images inside the same repository (this is the current approach, I think). It's not beautiful, but our license allows just 50 repository so using 3 for one single website seems like a waste.

Reply:
You're right. I will change the yaml key name.

As for the comment, I don't think we have a limit on the number of public repositories.

Reply:
You're right! Definitely better in separate repos then :)